app-information.html

```diff
@@ -0,0 +1,369 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>
+      Web App Manifest - App Information
+    </title>
+    <script src="https://www.w3.org/Tools/respec/respec-w3c" class=
+    "remove"></script>
+    <script class='remove'>
+    var respecConfig = {
+      mdn: true,
+      previousPublishDate: "2013-12-17",
+      specStatus: "NOTE",
+      shortName: "appmanifest-appinfo",
+      prevVersion: "",
+      previousMaturity: "",
+      formerEditors: [
+        {
+          name: "Rob Dolin",
+          company: "Microsoft Corporation",
+          companyURL: "https://www.microsoft.com/en-us/",
+        },
+      ],
+      editors: [
+        {
+          name: "Aaron Gustafson",
+          company: "Microsoft Corporation",
+          companyURL: "https://microsoft.com/",
+          w3cid: 43672,
+        },
+      ],
+      wg: "Web Applications Working Group",
+      wgURI: "https://www.w3.org/2019/webapps/",
+      wgPatentURI: "https://www.w3.org/2004/01/pp-impl/114929/status",
+      github: "https://github.com/w3c/manifest/",
+      caniuse: {
+        feature: "web-app-manifest-appinfo",
+        browsers: [
+          "chrome",
+          "firefox",
+          "safari",
+          "edge",
+          "and_chr",
+          "and_ff",
+          "ios_saf",
+        ],
+      },
+      xref: "web-platform",
+    };
+    </script>
+    <style>
+      .icon-title {
+        text-transform: uppercase;
+        font-weight: bold;
+      }
+
+      .icons {
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+      }
+
+      .icons > figure {
+        text-align: left;
+        margin: 0;
+        padding: 10px;
+        width: 188px;
+      }
+
+      .icons > figure > img {
+        width: 188px;
+        height: 188px;
+        min-width: 188px;
+        min-height: 188px;
+      }
+
+      .icons > figure .icon-title {
+        display: block;
+      }
+    </style>
+  </head>
+  <body data-cite="ENCODING SCREEN-ORIENTATION">
```

Comment:
Don't need these anymore... 

```suggestion
  <body>
```